CHANGELOG.md

```diff
@@ -12,6 +12,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add traits to reuse `Blockchain`s across multiple wallets (`BlockchainFactory` and `StatelessBlockchain`).
 - Upgrade to rust-bitcoin `0.28`
```

Comment:
```suggestion
- Upgrade to rust-bitcoin `0.28.1`
```
Looks like this rust-bitcoin version is only to enable higher MSRV for no_std CI tests. But still a good idea to do and update here in readme.